packages/framework/src/Migrations/Version20240827110317.php

```diff
@@ -18,13 +17,15 @@ class Version20200129140317 extends AbstractMigration implements ContainerAwareI
      */
     public function up(Schema $schema): void
     {
-        foreach ($this->getAllDomainIds() as $domainId) {
-            $this->sql(
-                'INSERT INTO "setting_values" ("name", "domain_id", "value", "type") VALUES 
+        if ($this->isAppMigrationNotInstalledRemoveIfExists('Version20200129140317')) {
+            foreach ($this->getAllDomainIds() as $domainId) {
+                $this->sql(
+                    'INSERT INTO "setting_values" ("name", "domain_id", "value", "type") VALUES 
                     (\'transferDaysBetweenStocks\', :domainId, \'1\', \'integer\'),
                     (\'deliveryDayOnStock\', :domainId, \'1\', \'integer\')',
```

Comment:
`deliveryDayOnStock` was already removed in https://github.com/shopsys/shopsys/commit/b77f53b7d58659ee95f2e3628af23a9b533778fb so I would not include it in this migration (it would create a "dead" entry in the `setting_values` table)